kustomize/install/rbac/cluster/role.yaml

```diff
@@ -3,8 +3,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: postgres-operator
-  labels:
-    postgres-operator.crunchydata.com/control-plane: postgres-operator
```

Comment:
Part of the difficulty of running two installers was that commonLabels was applied to everything -- as you would expect -- but what was set in the parent `kustomization.yaml` would override any child.

Anyway, long way of saying: I think everything that is part of the control-plane should have that label because that makes it easy for a user to filter out what's part of this installation and what isn't.

Now if we all agree that everything should have that label, how do we apply that? Right now, the Helm install uses the template `install.clusterLabels` to add that to all the k8s objects (I think); for kustomize, I think we should go ahead and use the commonLabels (in `kustomize/install/default/kustomization.yaml` and `kustomize/install/singlenamespace/kustomization.yaml`).

What do you think @cbandy ? If we only have one pod (and associated RBAC) for pgo, should everything have the control-plane label?

Reply:
> should everything have the control-plane label?

Yes, please.

> Now if we all agree that everything should have that label, how do we apply that?

I'm okay with anything. Here, I recommend the smallest/easiest change to achieve the desired effect.

🌱 Our minimum versions have moved far enough that I think we can rely on Kustomize v4.1 features now. Outside of this PR, we can investigate the behavior of [`labels`](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/labels/), for example.

Reply:
I've added `postgres-operator.crunchydata.com/control-plane: pgo` to commonLabels.

Reply:
@cbandy @benjaminjb Adding `postgres-operator.crunchydata.com/control-plane: pgo` to commonLabels brings kustomize into alignment here, but out of alignment with the installer in operator, which sets the key to `postgres-operator`. Is there a preference?

Reply:
We discussed this in meeting and have decided to leave as is for now.